### PR TITLE
[SCL 20] Task List Page

### PR DESCRIPTION
**Pull request by emilyw26**
* Uses mock data for the Task List Pages (manhattan is fleshed out, will add more to brooklyn and queens)
* Adds background image for app
* Styles the page according to Figma

https://user-images.githubusercontent.com/16674015/232181431-3387ae54-4d16-473e-8545-65d711d12249.mov

### CODE REVIEW

File: app/src/App.tsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 import {
-  getAuth,
-  User,
-  signInWithPopup,
-  GoogleAuthProvider,
-  sendSignInLinkToEmail,
-  isSignInWithEmailLink,
-  signInWithEmailLink,
-  signOut,
+    getAuth,
```
Comment (emilyw26) [inline app/src/App.tsx:6]: my IDE automatically added this formatting in this file

Comment (stanzheng) [reply app/src/App.tsx:6]: we can add some eslint rules later

**Comment by stanzheng** [comment]
this looks like the routes we have on-hand, LGTM and the googly eyes are great 

         <Route path="/" element={<HomePage user={user}/>}/>
          <Route path="/login" element={<LoginPage />}/>
          <Route path="/tasks/manhattan" element={<TaskList />}/>
          <Route path="/tasks/brooklyn" element={<TaskList />}/>
          <Route path="/tasks/queens" element={<TaskList />}/>
          <Route path="/tasks/manhattan" element={

**Review by stanzheng — APPROVED** [review]
LGTM like the googly eyes

**Pull request merged**